client/src/www/views/licenses_view/licenses/licenses.txt

```diff
@@ -6349,7 +6349,6 @@ SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 The following npm package may be included in this product:
 
- - cordova-plugin-splashscreen@6.0.2
```

Comment:
We should remove the whole block, not just this line. But the best is to regenerate this file. There are instructions somewhere.